relay/helper/stream_scanner_test.go

```
@@ -336,6 +336,60 @@ func TestStreamScannerHandler_PingSentDuringSlowUpstream(t *testing.T) {
+	oldEnabled := setting.PingIntervalEnabled
+	oldSeconds := setting.PingIntervalSeconds
+	setting.PingIntervalEnabled = true
+	setting.PingIntervalSeconds = 1
+	t.Cleanup(func() {
+		setting.PingIntervalEnabled = oldEnabled
+		setting.PingIntervalSeconds = oldSeconds
+	})
+
+	pr, pw := io.Pipe()
+	go func() {
+		defer pw.Close()
```

golangci-lint (2.13.2)

[error] Error return value of `pw.Close` is not checked

(errcheck)